source/account_structure/set_up_where_services_link_to/index.html.md.erb

```diff
@@ -0,0 +1,42 @@
+---
+title: Set up how your services link to GOV.UK Pay services
+last_reviewed_on: 2021-02-10
+review_in: 6 months
+weight: 142
+---
+
+# Set up how your services link to GOV.UK Pay services
+
+Each GOV.UK Pay service can take payments from one or more of your services.
+
+GOV.UK Pay services are free, and there's no limit to how many you can create.
+
+## Link your services to a single GOV.UK Pay service
+
+You can use one GOV.UK Pay service to take payments for a group of your services, if you need to easily manage the following across the whole group:
+
+- your staff members' access in the GOV.UK Pay admin tool
+- your account settings, for example which card types you accept
+
+1. [Create one GOV.UK Pay service](https://selfservice.payments.service.gov.uk/my-services/create).
+2. Use API keys or payment links from the GOV.UK Pay service in all of the services in your group.
+
+You will not be able to give your services different names or send payments to different bank accounts.
```

Comment:
You can change the description the user sees when making the payment. 

(I don't think we make ti very clear in the docs or product pages where all the infrmation you enter ends up on the payment page - ie service name is at the top, description is at the side. not for this PR but something to consider elsewhere....)